MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -20,12 +20,18 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorStateChange: Observer<String?>?
 
 	func loadFeed() {
 		onLoadingStateChange?(true)
+		onErrorStateChange?(nil)
 		feedLoader.load { [weak self] result in
-			if let feed = try? result.get() {
+			switch result {
+			case .success(let feed):
 				self?.onFeedLoad?(feed)
+				self?.onErrorStateChange?(nil)
```

Since the error is hidden on line 27, we don't need to hide it againon line 32. You can remove this line and the tests still pass:

```suggestion
```

---

done